incubator/hnc/internal/reconcilers/hierarchy_config.go

```diff
@@ -267,6 +274,33 @@ func (r *HierarchyConfigReconciler) syncExternalNamespace(log logr.Logger, nsIns
 	ns.ExternalTreeLabels = etls
 }
 
+// upgradeSubnamespaceAnnotation replaces the deprecated "subnamespaceOf"
+// annotation with the new "subnamespace-of" annotation. If both exist, we will
+// only delete the deprecated annotation.
+func (r *HierarchyConfigReconciler) upgradeSubnamespaceAnnotation(inst *corev1.Namespace) {
+	sOf, exist := inst.GetAnnotations()[api.DeprecatedSubnamespaceOf]
```

Comment:
This method interacts with `inst`'s annotations in three separate ways - once in an expression (this line), once as a temporary variable (line 287) and once through a helper function (line 300). This makes it quite difficult to follow the flow of information. Do you think you could find a way to make these all look more similar?

Reply:
Done.